<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleGUI</class>
 <widget class="QDialog" name="ModuleGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Module</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="Title">
     <property name="text">
      <string>Name of module</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Filetype">
     <property name="text">
      <string>File type :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Directory">
     <property name="text">
      <string>Directory :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="MSize">
     <property name="text">
      <string>Mapped size :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="DSize">
     <property name="text">
      <string>Disk size :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="CreationTime">
     <property name="text">
      <string>Creation time :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="LasteWriteTime">
     <property name="text">
      <string>Last write time :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="LastAcessTime">
     <property name="text">
      <string>Last acess time :</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
